packages/jsii-rosetta/bin/jsii-rosetta.ts

```diff
@@ -166,6 +168,55 @@ function main() {
         }
       }),
     )
+    .command(
+      'transliterate [ASSEMBLY..] [LANGUAGE..]',
+      '(EXPERIMENTAL) Transliterates the designated assemblies to the specified language (or all supported languages)',
+      (command) =>
+        command
+          .positional('ASSEMBLY', {
+            type: 'string',
+            string: true,
+            default: new Array<string>(),
+            required: true,
+            describe: 'Assembly to transliterate',
+          })
+          .positional('LANGUAGE', {
+            type: 'string',
+            string: true,
+            default: new Array<string>(),
```

Comment:
Is this supposed to also be an array?

Reply:
Yeah, you can submit several languages if you want to. Interestingly, I don't know how it would work (since it's a sequence of string[] positional arguments, so how'd it tell when it's one and when it's the other? I gotta fix that).